docs/notebooks/01_merge.ipynb

```diff
@@ -1,27 +1,505 @@
 {
```

Comment:
<p>It is also memory-inefficient to a point it will be an issue for large datasets. </p><p> </p><p>you can operate on the <code>.data</code> attribute of a CSR matrix: </p><p><code></code><code>python</p><p>adata.X.data = np.ceil(adata.X.data).astype(int)</p><p></code><code></code></p><p> </p>